src/components/charts/SensorDetailsBarChart.tsx

```diff
@@ -0,0 +1,108 @@
+/* eslint-disable react/require-default-props */
+import { ChartData, ChartOptions } from "chart.js";
+import CountSensorSchema from "../models/readings/CountSensorSchema";
+import SensorReadingSchema from "../models/readings/SensorSchema";
+import { getFormattedCountData } from "../presentation/uiHelpers";
+import { CHART_DATE_FORMAT } from "./chartHelper";
+import BarChart from "./BarChart";
+
+type SensorDetailsBarChartProps = {
+  label: string;
+  chartColor: string;
+  data: {
+    when: string;
+    value: number;
+  }[];
+  schema: SensorReadingSchema<number>;
+};
```

Comment:
Since this is identical to the props for the line chart maybe give it a more generic name and put it in the `chartHelper`?

Reply:
Nice catch. I renamed it to `SensorDetailsChartProps` and moved it to `chartHelper` as suggested.